acceptance/tests/subcommands/init.rb

```diff
@@ -7,7 +7,8 @@ def delete_root_folder_contents
 
   step 'ensure beaker init writes YAML configuration files to disk' do
     delete_root_folder_contents
-    on(default, 'beaker init')
+    result = on(default, 'beaker init --hosts centos6-64')
+    assert_match(/Trying as beaker-hostgenerator input/, result.stdout)
```

Comment:
I'm not clear why this assert is here; it seems to be testing that beaker is trying input as a hostgenerator string, but that is not what this step is trying to test; it's ensuring that YAML files are outputted to disk on the SUT. It seems like this assert is testing the implementation not pertinent to subcommand. 

A better test might be to read in the subcommand_options and ensure that the HOSTS object has a single element in the array.